roles/confluent.control_center/tasks/main.yml

```
@@ -14,6 +14,15 @@
     - ansible_os_family
     - ansible_fqdn
 
+- name: Stop Service and Remove Packages
```

```suggestion
- name: Stop Service and Remove Packages on Version Change
```

This task name is a little alarming :)

---

Fixed; here and in all other component tasks.